setup.cfg

```diff
@@ -18,7 +18,10 @@ multi_line_output = 3
 ignore_missing_imports = true
 strict_optional = true
 
-[mypy-openslides.utils.auth]
+[mypy-openslides.utils.dispatch]
+ignore_errors = true
```

Comment:
Which errors remain?

Reply:
This file is a bit complicated and I do not understand it. As I wrote above, I would like to remove it. Therefore I don't want to spend any time on it.
But of cause, you can add typing information as you like